manifests/config.pp

```diff
@@ -245,14 +244,14 @@
     notify => Class['keycloak::service'],
   }
 
-  file { "${keycloak::install_base}/${_subdir}/configuration":
+  file { ${_server_conf_dir}:
```

Comment:
You would need to define `$_server_conf_dir` for when `$::keycloak::operating_mode != 'domain'`.